CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Version [3.3.1] - (2019-04-25)
```

Comment:
Additionally the link to `[3.3.1]` does not exist. Can you add it to the end of this file (https://github.com/contentful/contentful-management.java/pull/126/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR172)?